docs/en/observability/uptime-set-up.asciidoc

```diff
@@ -0,0 +1,107 @@
+[[uptime-set-up]]
+= Set up monitors
+
+++++
+<titleabbrev>Set up monitors</titleabbrev>
+++++
+
+To set up an uptime monitor, you will pull an Elastic-provided Docker image, configure the monitor,
+and use the Docker image to run the monitor and send data back to Elasticsearch.
+After setup is complete, the data will be available in the {uptime-app} in {kib} to view, analyze, and alert on.
+
+[discrete]
+[[uptime-set-up-choose]]
+== Choose an approach
+
+There are two ways to set up an uptime monitor: *{heartbeat}* or *{agent}*.
+
+[discrete]
+[[uptime-set-up-choose-heartbeat]]
+=== Heartbeat
+
+{heartbeat} is a lightweight daemon that you install on a remote server to periodically
+check the status of your services and determine whether they are available.
+Performance data is then gathered and reported back to {heartbeat},
+where it is formatted and sent to the {stack}.
+
+// What are the pros and cons?
+Using {heartbeat} works well when...
+
+The downsides of using {heartbeat} are...
```

Comment:
What are the benefits and downside of using Heartbeat?

Reply:
Heartbeat is great for administrators wanting to configure everything via YAML files. In the future we'd like to provide easy ways to do this without using heartbeat directly, but that is currently the case, and something that will be supported for at least the next 2-3 years.

The downside of using heartbeat is that it requires more sysadmin type knowledge, users comfortable with a GUI won't be able to use it most likely.